src/site/asn4.rst

```diff
@@ -145,19 +145,58 @@ sorted order as laid out in ``sorted``, then the ``HuffmanNodeComparator`` shoul
 Part 4 --- Complete ``HuffmanCode``
 ===================================
 
+The ``HuffmanCode`` class will make use of the ``HuffmanNode`` and ``HuffmanNodeComparator`` class to implement the
+compression. Ultimately, this class will be responsible for building the Huffman Tree based on some seed string and
+encoding and decoding strings.
 
 
 Notes About Your Implementation
 -------------------------------
 
+* You will want to use a ``PrioriryQueue``, but instead of implementing one on your own, `this time you will use the one
+from java.util. <https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/PriorityQueue.html>`_
+    * `Make use of the constructor that takes a comparator <https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/PriorityQueue.html#%3Cinit%3E(java.util.Comparator)>`_
+
+* You will be making use of factory methods for instantiating the objects
+    * Refer to the ``Maze`` class from assignments 2 and 3
+    * I have provided you with an already complete ``fromFile`` factory method
```

Comment:
This is a little misleading, since the implementation delegates to `fromString` which is not implemented.